gtsfm/utils/io.py

```diff
@@ -151,9 +154,19 @@ def write_images(gtsfm_data: GtsfmData, save_dir: str) -> None:
     """
     os.makedirs(save_dir, exist_ok=True)
 
+    num_imgs = gtsfm_data.number_images()
+    # TODO: compute this
+    mean_obs_per_img = 0
+
+    # TODO: compute this
+    img_fname = "dummy.jpg"
+
     file_path = os.path.join(save_dir, "images.txt")
     with open(file_path, "w") as f:
-        f.write("# Number of cameras: {}\n".format(gtsfm_data.number_images()))
+        f.write("# Image list with two lines of data per image:")
+        f.write("#   IMAGE_ID, QW, QX, QY, QZ, TX, TY, TZ, CAMERA_ID, NAME")
```

Comment:
Technically, camera ID and image ID can be different. How will we handle that? Maybe we need the camera-id image-id map in our pipeline too.

Reply:
i think that is a more conceptual change that will require some re-thinking. I added it to an issue https://github.com/borglab/gtsfm/issues/130